helpers/patterns.js

```diff
@@ -1,5 +1,5 @@
 const regexPatterns = {
-	email: /^[a-z0-9!'#$%&*+\/=?^_`{|}~-]+(?:\.[a-z0-9!'#$%&*+\/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-zA-Z]{2,}$/i,
```

Comment:
these slashes after the escaping character should be reinserted

Reply:
@MaizeHH I'm pretty sure this is your groups fixes. I sent you like a month ago some linting errors I got and you fixed it and pushed. However, it was a mistake so they were pushed to staging immidately, and therefore, someone overwrote the changes. So this is actually a copy from your group commit (I think Morten).

Reply:
see commit fc6eff1

Reply:
Ok 👍 we'll make a hotfix for it